misago/sso/user.py

```diff
@@ -8,19 +8,24 @@
 def get_or_create_user(request, user_data):
     try:
         user = User.objects.get(sso_id=user_data["id"])
-        if user_needs_updating(user, user_data):
-            update_user(user, user_data)
+        # if user_needs_updating(user, user_data):
```

Comment:
Not a fan of `user = smth` followed by `return user` in next line.

Is there a reason for commenting out code instead of deleting it? If `user_needs_updating` is no longer used, we should delete it IMHO.